pkg/engine/events.go

```diff
@@ -70,14 +70,15 @@ type DiagEventPayload struct {
 
 // PolicyViolationEventPayload is the payload for an event with type `policy-violation`.
 type PolicyViolationEventPayload struct {
-	ResourceURN       resource.URN
-	Message           string
-	Color             colors.Colorization
-	PolicyName        string
-	PolicyPackName    string
-	PolicyPackVersion string
-	EnforcementLevel  apitype.EnforcementLevel
-	Prefix            string
+	ResourceURN          resource.URN
+	Message              string
+	Color                colors.Colorization
+	PolicyName           string
+	PolicyPackName       string
+	PolicyPackVersion    string
+	EnforcementLevel     apitype.EnforcementLevel
+	Prefix               string
+	PolicyPackVersionTag string
```

Comment:
Nit: Would we want to move this field up, so that it is next to the other `PolicyPack*` fields? Would it make sense to avoid the repetition and create a `type PolicyPackInfo struct` to bundle the data?

Same for other structs where we add the new `PolicyPackVersionTag` field, like `AnalyzeDiagnostic`.

Reply:
I ended up removing it from a lot of places. But I will double check these.. The reason I put it at the bottom was because this mimics the protobuf schema which is numbered and since we dont have any empty numbers, I had to put it at the end to remain backwards compatible...

My preference is that this stays in the same order at the protobuf so we keep consistent order.